ckanext/spatial/commands/spatial.py

```diff
@@ -31,67 +32,30 @@ class Spatial(CkanCommand):
 
     summary = __doc__.split('\n')[0]
     usage = __doc__
-    max_args = 2 
+    max_args = 2
     min_args = 0
 
     def command(self):
         self._load_config()
-        print ''
+        print('')
```

Comment:
do we need this line ?!

Reply:
This dates back [>10 years](https://github.com/ckan/ckanext-spatial/commit/1710205cf9f07c3d5cacd9ecf66c04e210701ba3#diff-81ab1f39620ac260630bf2b36495db3ae52983a098268963ff677b8e2f3e6e03R32)! I guess `load_config()` printed some logging, and this new-line made it neat. I don't know if it's still needed.

Reply:
I will delete this line.

Reply:
Done.